docs/one-click-integrations.md

```diff
@@ -28,7 +28,7 @@ Pulse integrates directly with GitHub to receive data about deployments and chan
 !!! important
     Consider the following before using the GitHub integration:
 
-    -   The integration obtains information about deployments from Git tags that follow the [SemVer](https://semver.org) convention, excluding pre-release versions.
+    -   The integration obtains information about deployments from Git tags that follow the [SemVer](https://semver.org) convention, excluding pre-release versions, but allowing release prefixes. Valid tags e.g.: `1.0.0`, `v2.3.4`
```

Comment:
Small suggestion to simplify the punctuation:

```suggestion
    -   The integration obtains information about deployments from Git tags that follow the [SemVer](https://semver.org) convention, excluding pre-release versions but allowing release prefixes. For example, `1.0.0` or `v2.3.4`.
```

I really like the idea of providing examples! :+1: